src/app/compliance-rule-configuration/compliance-rule-configuration.component.html

```diff
@@ -0,0 +1,45 @@
+<div>
+  <mat-form-field appearance="fill" >
+    <mat-label>Create new Configuration from Rule:</mat-label>
+    <mat-select [(value)]="selectedRule">
+      <mat-option *ngFor="let complianceRule of complianceRules" [value]="this.utils.getId(complianceRule)">
+        Rule {{this.utils.getId(complianceRule)}}
+      </mat-option>
+    </mat-select>
+  </mat-form-field>
+
+  <br>
+  IssueType:
+  <input matInput style="outline-width: 3px; outline-color: black" type="text" [(ngModel)]="this.issueType"/>
+
+  <button mat-button (click)="createComplianceRuleConfiguration(selectedRule);">ADD</button>
+
+  <mat-form-field appearance="fill" >
+    <mat-label>Add available Configuration:</mat-label>
```

Comment:
I don't think this makes really sense... The relation between a compliance job and a rule configuration is one-to-many not many-to-many. Therefore, there is no source for these "existing" rule configurations, unless you want to reassign a rule configuration from another job to this job, but I don't see why you would do that...

Reply:
In general, I'm not sure why it would be bad to be able to do that, but going to remove it

Reply:
I think it could be a problem to reassign Rule Configurations from on job to another because the original job (which was probably correctly configured) doesn't have these rule configurations anymore. I think configuring a job should not affect other jobs.